cargo-shuttle/src/lib.rs

```diff
@@ -545,62 +553,72 @@ mod tests {
     }
 
     #[test]
-    fn secrets_file_is_archived() {
+    fn make_archive_include_secrets() {
         let working_directory =
             canonicalize(path_from_workspace_root("examples/rocket/secrets")).unwrap();
 
-        let mut secrets_file = File::create(working_directory.join("Secrets.toml")).unwrap();
-        secrets_file
-            .write_all(b"MY_API_KEY = 'the contents of my API key'")
-            .unwrap();
+        fs::write(
+            working_directory.join("Secrets.toml"),
+            "MY_API_KEY = 'the contents of my API key'",
+        )
+        .unwrap();
 
-        let mut project_args = ProjectArgs {
+        let project_args = ProjectArgs {
             working_directory,
             name: None,
         };
 
-        let mut shuttle = Shuttle::new().unwrap();
-        shuttle.load_project(&mut project_args).unwrap();
+        let entries = get_archive_entries(project_args);
 
-        let file = shuttle.run_cargo_package(true).unwrap();
+        assert_eq!(
```

Comment:
Also CI is failing because of a mismatch here, which looks to be from the ordered comparison and it seems the walk ordering is not deterministic.